tests/sentry/debug/utils/test_exception_reporter_filter.py

```diff
@@ -0,0 +1,20 @@
+from unittest import mock
+
+from django.test import override_settings
+from django.urls import reverse
+
+from sentry.testutils.cases import TestCase
+from sentry.web.frontend.error_500 import Error500View
+
+
+@override_settings(DEBUG=True)
+@override_settings(DEBUG_PROPAGATE_EXCEPTIONS=False)
+@override_settings(ROOT_URLCONF="sentry.conf.urls")
+class TestNoSettingsInDebugView(TestCase):
```

Comment:
I added this test to ensure the intended behaviour of `get_safe_settings` -- it was failing before I added the new settings filterer